docs/guides/vault/edit-vault-data.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_label: 'Edit Vault Data'
+sidebar_position: 2
+---
+
+# Edit Vault Data
+
+This guide will teach you how to set data to an **[LSP9Vault](../../standards/smart-contracts/lsp9-vault.md)** contract. Any data can be attached to the vault and since it supports **[LSP1-UniversalReceiver](../../standards/generic-standards/lsp1-universal-receiver.md)** standard, we will be setting the [**Universal Receiver Delegate**](../../standards/smart-contracts/lsp1-universal-receiver-delegate-vault.md) address inside the storage.
+
+## Setting Data (Universal Receiver Delegate)
+
+The default implementation of the **Universal Receiver Delegate** of the Vault, that we are going to deploy will register the assets received to the storage and and will remove them when their balance equals 0.
+
+### Deploying Universal Receiver Delegate (URD)
+
+:::info
+
+The **Universal Profile** and the **Vault** don't use the same implementation of the Universal Receiver Delegate.
+:::
+
+```typescript title="Deploying the universal receiver delegate of the vault"
+import LSP1UniversalReceiverDelegateVault from '@lukso/lsp-smart-contracts/artifacts/LSP1UniversalReceiverDelegateVault.json';
+
```

Comment:
As for the create vault comment, could you also add the web3 import, etc. here so we can simply copy / paste this code snippet and it works?